examples/gke-a4/nccl-installer.yaml.tftpl

```diff
@@ -25,6 +25,8 @@ spec:
       k8s-app: nccl-rdma-installer
   updateStrategy:
     type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 16
```

Comment:
Curious to know why this is 16

Reply:
Somewhat arbitrary. 1 (the current/default value) is extremely slow. Approximately one order of magnitude seems like a good improvement, but two orders of magnitude might potentially strain networking resources. So, I went for a multiple of 8 (given typical block sizes) that was about one order of magnitude.